zulipterminal/model.py

```diff
@@ -885,6 +885,17 @@ def is_muted_topic(self, stream_id: int, topic: str) -> bool:
         topic_to_search = (stream_name, topic)
         return topic_to_search in self._muted_topics
 
+    def get_current_topic_from_message(
```

Comment:
Minor (naming):
- Re naming, the 'current' has no meaning when reading this independently.
- Based on an argument in another comment, this could be `stream_topic_from_message_id`?